lib/cssom.js

```diff
@@ -11,12 +11,22 @@
 declare class StyleSheet {
   disabled: boolean;
   href: string;
+  media: MediaList;
   ownerNode: Node;
   parentStyleSheet: ?StyleSheet;
   title: string;
   type: string;
 }
 
+declare class StyleSheetList {
+    length: number;
+    [index: number]: StyleSheet;
+}
+
+declare class MediaList {
+
```

Comment:
Unfortunately not yet documented on MDN: https://developer.mozilla.org/en-US/docs/Web/API/MediaList